portfolio-project-matching-app/components/DBForm.js

```diff
@@ -0,0 +1,47 @@
+import { addNewDoc } from '../Firebase/clientApp.ts'
+
+const DBForm = () => {
```

Comment:
`DBForm.js` doesn't really add anything to the project, we can get rid of it at any time. However, it does provide an easy way to tinker around with Firebase data.